partner-chains-cli/src/pc_contracts_cli_resources.rs

```diff
@@ -20,18 +20,22 @@ impl Default for PcContractsCliResources {
 				hostname: KUPO_HOSTNAME.default.unwrap_or("localhost").to_string(),
 				port: KUPO_PORT.default.unwrap_or("1442").parse().unwrap(),
 			},
-			ogmios: ServiceConfig {
-				protocol: OGMIOS_PROTOCOL
-					.default
-					.and_then(|p| NetworkProtocol::from_str(p).ok())
-					.unwrap_or(NetworkProtocol::Http),
-				hostname: OGMIOS_HOSTNAME.default.unwrap_or("localhost").to_string(),
-				port: OGMIOS_PORT.default.unwrap_or("1337").parse().unwrap(),
-			},
+			ogmios: default_ogmios_service_config(),
```

Comment:
register1.rs needs only Ogmios, so I extracted Ogmios part only. I think that Kupo part will be removed (and this file renamed) when we stop using pc-smart-contracts cli.